luggage.make

```diff
@@ -343,7 +352,7 @@ kill_relocate:
 	@-sudo /usr/bin/python -c "$${PYTHON_PLISTER}"
 
 local_pkg:
```

Comment:
Delete this target; it's not being called anywhere as you added these commands to compile_package_*